src/apis/accesscontextmanager/v1beta.ts

```diff
@@ -449,19 +445,6 @@ export namespace accesscontextmanager_v1beta {
      */
     enableRestriction?: boolean | null;
   }
-  /**
-   * Alpha. Specifies how APIs are allowed to communicate within the Service Perimeter. This message is DEPRECATED and had been renamed to VpcAccessibleServices
-   */
-  export interface Schema$VpcServiceRestriction {
```

Comment:
most of the updates below are the result of running the generator, which tends to change a few existing APIs, regardless of changes to the templates.